.semaphore/publish.yml

```diff
@@ -1,12 +1,9 @@
 version: v1.0
-
```

Comment:
Are these changes intentional?

Reply:
Was the configuration done in the Semaphore web UI?

We should make the changes in the YAML file directly to avoid unnecessary changes.